src/Microsoft.IdentityModel.Tokens/CompatibilitySuppressions.xml

```diff
@@ -0,0 +1,10 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- https://learn.microsoft.com/en-us/dotnet/fundamentals/package-validation/diagnostic-ids -->
+<Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <Suppression>
+    <DiagnosticId>CP0002</DiagnosticId>
+    <Target>M:Microsoft.IdentityModel.Tokens.JsonWebKeyConverter.ConvertFromECDsaSecurityKey(Microsoft.IdentityModel.Tokens.ECDsaSecurityKey)</Target>
```

Comment:
oh I think I see what the tool is trying to catch.

net461 implements netstandard 2.0

So this is a bit confusing. Is there any reason ConvertFromECDsaSecurityKey cannot also be included in net461?

Reply:
@keegan-caruso even though the api is available in netstandard 2.0, we found 461 and 472 did not support this api.
https://learn.microsoft.com/en-us/dotnet/api/system.security.cryptography.eccurve?view=netstandard-2.0
Some types are not supported.

Reply:
interesting!

Reply:
@keegan-caruso i wrote 472 above, but i meant 462.
What puzzles me is why only an error for 461 and not 462?